web/src/components/sign-up/sign-up.css

```diff
@@ -0,0 +1,55 @@
+@import url('../vars.css');
+
+.submission {
+    height: 45px;
+    display: flex;
+}
+
+input {
```

Comment:
Unfortunately our css import system doesn't namespace files, so these selectors apply globally. Can we namespace it under a className like .newsletter-sign-up?